tools/OutlineService/OutlineService/OutlineService.cs

```diff
@@ -224,6 +224,25 @@ public partial class OutlineService : ServiceBase {
       }
       eventLog.WriteEntry($"Got system gateway IP {systemGatewayIp.ToString()}");
 
+      // Set a low device metric for the TAP device, to ensure the system

```

Comment:
I think `the lowest possible` would be clearer than `a low`.

Reply:
Agreed. Done.